client/components/plans/plans-compare/index.jsx

```diff
@@ -86,59 +83,6 @@ var PlansCompare = React.createClass( {
 		page( plansLink );
 	},
 
-	showFreeTrialException: function() {
-		const hasTrial = this.props.selectedSite
-				? this.props.selectedSite.plan.free_trial
-				: false,
-			canStartTrial = this.props.sitePlans && this.props.sitePlans.hasLoadedFromServer
-				? this.props.sitePlans.data.some( property( 'canStartTrial' ) )
-				: false;
-
-		if ( getABTestVariation( 'freeTrials' ) !== 'offered' ) {
-			return false;
-		}
-
-		// always show if the user is currently in trial
-		if ( hasTrial ) {
-			return true;
-		}
-
-		// show if the site is eligible for a trial (it never had a free trial before)
-		// and free trial is enabled for this component
-		if ( canStartTrial && this.props.enableFreeTrials ) {
-			return true;
-		}
-
-		// show if we are in signup and free trial is enabled for this component
-		if ( this.props.isInSignup && this.props.enableFreeTrials ) {
-			return true;
-		}
-
-		return false;
-	},
-
-	freeTrialExceptionMarker: function( feature ) {
-		if ( this.showFreeTrialException() && featuresListUtils.featureNotPartOfTrial( feature ) ) {
-			return '*';
-		}
-
-		return null;
-	},
-
-	freeTrialExceptionMessage: function() {
-		if ( ! this.isDataLoading() &&
-			this.showFreeTrialException() &&
-			this.getFeatures().some( featuresListUtils.featureNotPartOfTrial ) ) {
-			return (
-				<div className="plans-compare__free-trial-exception-message">
```

Comment:
We should remove this `plans-compare__free-trial-exception-message` class from the stylesheet as well.